DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def plotImage(exposure: ExposureF, img_opt: dict = None):
</pre><p>Either remove "img_opt" (it isn't currently used) or use it in the function.</p><p> </p><p>Also, "opts" and "title" are not used, so don't need to be in docstrings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>